docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+#. Go through the Proctorio Setup for your Webcam, Desktop, etc. Upon setup completion, your webcam should appear to the right of the screen along with some controls. 
+#. On the "Return to exam" page, DO NOT click on the "Sending you back to your exam" page. Wait instead, and you should be sent to the "Important"/"Start Exam" interstitial automatically
+#. In a separate exam, go through the setup again and on the "Sending you back to your exam." page, click "Return to Exam". Clicking this link should send you to another interstitial. (NOTE: Clicking this link is time-bound so make sure to click it fast)
+#. Clicking the "Return to Exam" button should direct you to a page titled "Important" w/ another header saying "Proctored Exam Rules", and a "Start Exam" Button
+
+
+Exams Dashboard
+===============
+Expected Behavior: Calls to the ACS Endpoint work, instructors can review/reset exam attempts, and expected downstream effects (e.g. w/ completion, credits, certificates, etc) of modifying an exam attempt's status occur
+NOTE: These instructions currently do not account for downstream effects involving a learner's credit requirement status. We currently have no plan to include these
+
+Dashboard UI Loads Exam Attempts
+--------------------------------
+#. Clicking on the "Special Exams" tab should load the *new* version of the exams dashboard
+#. Clicking on the "Review Dashboard" tab within the exams dashboard should open a dashboard owned by your selected procotoring software (e.g. a Proctortrack or Proctorio dashboard for reviewing exam attempt details)
+#. Clicking the "Select An Exam" dropdown should show the list of exams for the course
+#. Filtering by exam name in the exam selection dropdown should bring up the expected exams (e.g. querying "M4-M6" shows all the exams in that section)
+#. Selecting a specific exam (i.e. the ones you just took) should load the respective exam attempts for those exams
+#. The exam attempts should be in the expected status, e.g. successful attempts are submitted or verified, and the errored attempts are errored
+#. Filtering by username only should show exam attempts for that username
+#. Clicking the "Review Dashboard" link in the modal should load the review dashboard for your chosen Proctoring Software (e.g. Proctortrack or Proctorio)
+
+Can Reject/Verify "Review Required" attempts
+---------------------------------------------------
+INSTRUCTIONS: Go into another exam and be suspiscious. Block your webcam, leave for a few seconds, make strange noises, open a bunch of tabs, watch some youtube, open wikipedia, ask chatgpt for the answer to life, etc. Note you will need to do this a couple of times
+
+#. Submit your suspiscious exam attempt, and check the exams dashboard. Assuming you were suspiscious enough, you should see a row that has a status of "Review Required" AND has a "Review Required" button for your attempt
```

Comment:
Would it be faster and less vendor specific to go into the admin dashboard and set the attempt's status to `second_review_required`? This also takes a way the need to be "suspicious enough" and avoids relying on the vendor's review process, which we don't need to be testing here. This applies to all the test cases that rely on a review coming in.

Reply:
Added instructions to have this as an option to just retaking the exam